CONTRIBUTING.md

```diff
@@ -92,7 +92,7 @@ While far from comprehensive, following these guidelines will get the easy stuff
 
 ##### Style
 
-1. This project uses snowpack + webpack plugin to build and compile. **Code produced by the build must work across all supported browsers and devices - Chrome, FF, Safari, Edge, and IE 9-11**
+1. This project uses vite to build and compile. **Code produced by the build must work across all supported browsers and devices - Chrome, FF, Safari, Edge, and IE 9-11**
```

Comment:
I don't think anything is actually still using the browserslist.  We should talk to product about if we need to support ie11... it officially goes to unsupported by MS in 6 weeks, so if we can avoid the extra overhead and polyfills it would be nice.

Reply:
I just double checked and this got added when I merged a bunch of content from the contributing guide of the player. Based on the browserslist, I don't think that was the intent, so I'm going to remove it.